src/lib.rs

```diff
@@ -2,19 +2,20 @@ use futures::{join, try_join};
 use near_indexer::near_primitives::types::ShardId;
 use near_indexer::{IndexerExecutionOutcomeWithReceipt, IndexerShard, StreamerMessage};
 use std::time::Duration;
-use tokio::sync::mpsc::Receiver;
+use tokio::sync::mpsc::{Receiver};
```

Comment:
Sorry for this change
```suggestion
use tokio::sync::mpsc::Receiver;
```